domains/sticknologic.json

```
@@ -10,10 +10,11 @@
     "A": ["65.19.154.90"],
     "AAAA": ["2001:470:1:1ee::1002"],
     "MX": [
-      { "target": "sticknologic.is-a.dev", "priority": 0 }
+      "mx1.improvmx.com", "mx2.improvmx.com"
     ],
     "TXT": [
-      "v=spf1 ip4:65.19.154.90 ip6:2001:470:1:1ee::1002 ~all"
+      "v=spf1 include:spf.improvmx.com ~all",
+      "v=spf1 include:_spf.mailersend.net ip4:65.19.154.90 ip6:2001:470:1:1ee::1002 ~all"
```

```suggestion
      "v=spf1 include:spf.improvmx.com include:_spf.mailersend.net ip4:65.19.154.90 ip6:2001:470:1:1ee::1002 ~all"
```

You should consolidate these